react/features/base/media/components/native/VideoTransform.js

```diff
@@ -189,21 +183,17 @@ class VideoTransform extends Component<Props, State> {
             onShouldBlockNativeResponder: () => false,
             onStartShouldSetPanResponder: this._onStartShouldSetPanResponder
         });
-
-        const { streamId } = this.props;
-
-        this._restoreTransform(streamId);
     }
 
     /**
-     * Implements React Component's componentWillReceiveProps.
+     * Implements React Component's componentDidUpdate.
      *
      * @inheritdoc
      */
-    componentWillReceiveProps({ streamId: newStreamId }) {
-        if (this.props.streamId !== newStreamId) {
-            this._storeTransform();
-            this._restoreTransform(newStreamId);
```

Comment:
I have a question about the old code (which isn't changed in my refactoring). When props has a new stream but there is no matching transform in the prop _transforms, the transform state of this component is not updated. Is that intentional?

Reply:
@zbettenbuk hopefully knows

Reply:
Yes, it's true. For sanity, I'll add this step, but it's not causing an issue now, because only one can share desktop at a time, if I'm right.

Thanks for pointing it out!